README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# COVID Stats
+# COVID Statistics
 
```

Comment:
- [ ] As a requirement you need to deploy a live version for your project, kindly fix the correct GitHub Pages URL for your live demo link, the one there is redirecting to a 404 page.

Reply:
Thanks for the feedback @danielufeli , Let me do it right now.

Reply:
It has now been resolved. 😄